test_setup.clouduser.yml

```diff
@@ -0,0 +1,81 @@
+---
+# This playbook adds new user account with sudo root privileges, based on user
+# account usually created by cloud-init[1] and disables direct root access via
+# ssh.
+#
+# This is usefull for testing ssh access in environment when we don't have
+# direct ssh root access on storage machines and need to connect via different
+# user and then use sudo instead.
+#
+# [1] https://cloudinit.readthedocs.io/en/latest/
+
+- hosts: all
```

Comment:
I wouldn't use `all` here, because this might be quite dangerous for example in CentOS CI, where is also `qe_server` part of the inventory file.
I would propose something like:
`- hosts: "usm_server:usm_nodes:usm_client"`